crates/tap-agent/src/tracker/sender_fee_stats.rs

```diff
@@ -9,20 +9,24 @@ use std::{
 use super::{AllocationStats, DefaultFromExtra, DurationInfo};
 use crate::{agent::unaggregated_receipts::UnaggregatedReceipts, backoff::BackoffInfo};
 
+/// Stats for a given allocation
 #[derive(Debug, Clone, Default)]
 pub struct SenderFeeStats {
+    /// sum of all fees in the tracker
     pub(super) total_fee: u128,
+    /// counter of receipts
     pub(super) count: u64,
-    // there are some allocations that we don't want it to be
-    // heaviest allocation, because they are already marked for finalization,
-    // and thus requesting RAVs on their own in their `post_stop` routine.
+    /// there are some allocations that we don't want it to be
+    /// heaviest allocation, because they are already marked for finalization,
+    /// and thus requesting RAVs on their own in their `post_stop` routine.
```

Comment:
```suggestion
    /// there are some allocations that we don't want to be the
    /// heaviest allocation. This is because they are already marked for finalization,
    /// and thus requesting RAVs on their own in their `post_stop` routine.
```